app/response_processor.py

```diff
@@ -102,6 +108,33 @@ def _encode(self, decrypted):
             raise QuarantinableError
         return xml
 
+    def _send_rm_receipt(self, case_id):
+        request_url = settings.RM_SDX_GATEWAY_URL
+
+        try:
+            r = self.session.post(request_url, auth=settings.BASIC_AUTH, json={'caseId': case_id})
+        except MaxRetryError:
+            logger.error("Max retries exceeded (5)",
+                         request_url=request_url,
+                         case_id=case_id)
+            raise RetryableError
```

Comment:
This seems a little odd, it's hit a max retry but then raises a retryable error.

Reply:
Yeah - unless we can sort out a way of using backoff retry in our queues in SDX then we just continue to POST to the endpoint, assuming that it will come back up at some time in the future.